pkg/ipmi/command.go

```diff
@@ -33,6 +33,10 @@ type Command interface {
 	// otherwise need to have a no-op layer created.
 	Operation() *Operation
 
+	// Most commands use RemoteLUN 0 (LUNBMC). Others (such as GetSensorReading
```

Comment:
Can you rearrange this to begin with `RemoteLUN`, so we follow the [convention](https://go.dev/blog/godoc#:~:text=Notice%20this%20comment%20is%20a%20complete%20sentence%20that%20begins%20with%20the%20name%20of%20the%20element%20it%20describes.)?